lib/open_api_spex/cast/utils.ex

```diff
@@ -0,0 +1,60 @@
+defmodule OpenApiSpex.Cast.Utils do
+  @moduledoc false
+
+  alias OpenApiSpex.Cast.Error
+
+  # Merge 2 maps considering as equal keys that are atom or string representation
+  # of that atom. Atom keys takes precedence over string ones.
+  def merge_maps(map1, map2) do
```

Comment:
Missing test.

Reply:
I've not implemented this and am not sure what the functions achieves. @albertored can you take a look?